core/src/main/scala/com/lightbend/paradox/ParadoxProcessor.scala

```diff
@@ -86,10 +87,12 @@ class ParadoxProcessor(reader: Reader = new Reader, writer: Writer = new Writer)
     lazy val getNavigation = writer.writeNavigation(pageToc.root(loc), context)
     lazy val hasSubheaders = page.headers.nonEmpty
     lazy val getToc = writer.writeToc(headerToc.headers(loc), context)
+    lazy val getSource_url = githubLink(Some(loc)).getHtml
```

Comment:
No, I think `$page.source_url$` looks ok. I wasn't sure if `getSource_url` is derived automatically. If it's going to be `$page.sourceUrl$`, I'm fine with `getSource_url`.